src/DataSet/ObjectDataSet.php

```diff
@@ -66,6 +66,11 @@ public function getAttributeValue(string $attribute): mixed
             : null;
     }
 
+    public function hasAttribute(string $attribute): bool
+    {
+        return true;
```

Comment:
I think we also need to check the value if it has the attribute as you did it ArrayDataSet.

Reply:
Agree. Also found another bug during making these changes. Will commit fix soon.

Reply:
Fixed.